.laminas-ci.json

```diff
@@ -6,23 +6,23 @@
         {
             "name": "Integration tests with memory adapter",
             "job": {
-                "command": "composer require laminas/laminas-cache-storage-adapter-memory && ./vendor/bin/phpunit --testsuite 'Laminas Cache Integration Tests'",
+                "command": "composer require laminas/laminas-cache-storage-adapter-apcu && ./vendor/bin/phpunit --testsuite 'Laminas Cache Integration Tests'",
```

Comment:
Could you please change the "name" of the Job(s) to "Integration tests with APCu adapter"